CONTRIBUTING.md

```diff
@@ -1,12 +1,204 @@
 # Contributing
 
-# Environment
+Wrangler2 is an open source project and we welcome contributions from you. Thank you!
 
-# Steps For Making Changes
+Below you can find some guidance on how to be most effective when contributing to the project.
+
+## Getting started
+
+### Set up your environment
+
+Wrangler2 is built and run on the node.js JavaScript runtime.
```

Comment:
Nit: `Node.js` is capitalized (not `node.js`).

Reply:
```suggestion
Wrangler2 is built and run on the Node.js JavaScript runtime.
```

Reply:
Right! Thanks.